### ISSUE TITLE
Add missing relabels for tags

### ISSUE THREAD

**Issue by eddieantonio** [UAlbertaALTLab/itwewina#36]
Continuing from #13.

Updating `itwewina.relabel`.

**Comment by aarppe** [comment]
Implemented (to a large part) in:

giella/apps/dicts/nds/src/neahtta/configs/language_specific_rules/user_friendly_tags/itwewina.relabel

Could add further tooltips for e.g. reduplication tags (referring to the iterative/intensive meanings they convey).

Further restructuring and cleaning up of the file will likely be in place.

**Comment by aarppe** [comment]
Plus implemented script for verification of associated root.lexc and *.relabel files, cf.

./verify-tag-relabeling.sh ~/giella/apps/dicts/nds/src/neahtta/configs/language_specific_rules/user_friendly_tags/itwewina.relabel ../../src/morphology/root.lexc | head -10
Duplicate tags in root LEXC file: ../../src/morphology/root.lexc
Duplicates (n=2) of tag: +Def on lines: 55 70
Duplicates (n=2) of tag: +Der/Dim on lines: 123 556
Duplicates (n=2) of tag: +Loc on lines: 47 553
Duplicates (n=2) of tag: +Obv on lines: 48 552
Duplicates (n=2) of tag: +Pl on lines: 112 551
Duplicates (n=2) of tag: +Px1Pl on lines: 118 561
Duplicates (n=2) of tag: +Px1Sg on lines: 114 558
Duplicates (n=2) of tag: +Px2Pl on lines: 120 563
Duplicates (n=2) of tag: +Px2Sg on lines: 115 559
...

Indicates a few obvious tags to be added for relabeling - the rest may need some cleaning of the FST of historical baggage and lazy duplicates.

**Comment by eddieantonio** [comment]
So is this issue satisfactorily addressed, or does it still need more work?

**Comment by aarppe** [comment]
It should be satisfactory for the time being, I just checked in relabelings for the few (potentially) missing tags - though some additional verification/info may be needed - but it's good for now.

On the longer run, we'd want to have a system where I (or another linguist) can check in modifications and have them incorporated in itwêwina (in the manner that was undertaken in the older admin code by running 'fab itwewina update_strings' or 'fab itwewina compile_strings').

**Comment by eddieantonio** [comment]
Cool! I'm getting an error in `paradigms.py`, but I think I know how to fix that:

```
IndexError
IndexError: Invalid tagset <dependency>. Choose one of: conjunct, animacy, prelemma_tags, pos, person, indicative, nds_dummy_tags, invisible_in_paradigms, tense, type, noun_animacy, preverbs
```

Once that's done, I'll close this issue!

**Comment by eddieantonio** [comment]
Fixed as of  6542cc6.

**Comment by aarppe** [comment]
Yup, I added the variable 'dependency' to the N-A-D and N-I-D *.paradigm specification files, but wasn't entirely sure whether I'd need to add that variable somewhere for things to work, presumably in the crk.tagset  file (which I hadn't ever touched).

If we get over-matching of layout files with *.paradigm files for nouns because of this, we might need to specify the 'dependency' variable for the N-A and N-I *.paradigm files as well (as being 'non-dependent').
